### PR TITLE
[DEL]To remove OCA standard author in EduERP module.

### PR DESCRIPTION
**Pull request by KrishnaPrajapati-SerpentCS**

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
fix conflicts plz. pull latest

File: barcode_report/report/result_label.py [hunk 1]
```diff
@@ -2,32 +2,52 @@
 # See LICENSE file for full copyright and licensing details.
 
 
-from odoo import models, api
-from odoo.report import report_sxw
+from odoo import api, fields, models
+from dateutil import parser
```
Comment (JayVora-SerpentCS) [inline barcode_report/report/result_label.py]: explain why fields and models?

Comment (JayVora-SerpentCS) [inline barcode_report/report/result_label.py]: Are we using parser?

File: barcode_report/report/result_label.py [hunk 2]
```diff
@@ -1,59 +1,47 @@
...
         }
-        standard_id = parser.parse(docargs.get('data').get('standard_id '))
-        docargs['data'].update({'standard_id': standard_id})
-        division_id = parser.parse(docargs.get('data').get('division_id '))
-        docargs['data'].update({'division_id': division_id})
-        medium_id = parser.parse(docargs.get('data').get('medium_id '))
-        docargs['data'].update({'medium_id': medium_id})
-        year_id = parser.parse(docargs.get('data').get('year_id '))
-        docargs['data'].update({'year_id': year_id})
         render_model = 'barcode_report.result_label'
```
Comment (JayVora-SerpentCS) [inline barcode_report/report/result_label.py]: Explain why these were removed?

File: exam/models/exam.py [hunk 3]
```diff
@@ -413,6 +413,7 @@ def compute_grade(self):
                             self.grade = grade_id.grade
     standard_id = fields.Many2one("school.standard", "Standard", required=True)
     year = fields.Many2one('academic.year', 'Academic Year', required=True)
+    grade_id = fields.Many2one('grade.master', "Grade")
     grade = fields.Char(_compute_='compute_grade', string='Grade', method=True,
                         store=True)
```
Comment (JayVora-SerpentCS) [inline exam/models/exam.py:418]: Do we need both of these fields?

**Comment by JayVora-SerpentCS** [comment]
Still there are conflicts

**Comment by KrishnaPrajapati-SerpentCS** [comment]
All conflicts are solved.

Not  shown any one conflict.

**Comment by KrishnaPrajapati-SerpentCS** [comment]
Solved All Conflicts

**Comment by JayVora-SerpentCS** [comment]
https://travis-ci.org/JayVora-SerpentCS/EduERPv8/jobs/187854930 fails

**Comment by JayVora-SerpentCS** [comment]
travis fails https://travis-ci.org/JayVora-SerpentCS/EduERPv8/builds/187858647

**Pull request merged**